### DISCUSSION TITLE
Error when trying to import JSON template

### DISCUSSION THREAD

**Discussion by tggaule** [monarc-project/MonarcAppFO#449] [Q&A]
Hello!

I don't really know if it's a bug or an error on my part but I would like to import a JSON template (https://github.com/monarc-project/monarc-objects/blob/master/referentials/ISO_IEC_27002.json)
but when importing it gives me a type error:
"An error has occurred on api/client-anr/1/instances/import : Typed property Monarc\FrontOffice\Service\InstanceImportService::$monarcVersion must not be accessed before initialization (500)"

Thanks a lot for your time on the project !

**To Reproduce**
1. Go to 'Project management context menu'
2. Click on 'Import a risk analysis'
3. Select JSON file
4. Click on 'Import file'
5. See error

**Expected behavior**
The template should be imported

**Screenshots**
![image](https://user-images.githubusercontent.com/83638055/180235617-0f8287e5-3099-4c33-a1ff-96c5b1a961d1.png)
![image](https://user-images.githubusercontent.com/83638055/180235929-6da28cb7-763f-445e-8086-34b9b34f93ef.png)
![image](https://user-images.githubusercontent.com/83638055/180235320-5a934bd7-0a92-4cb9-b75d-9c642983fcb3.png)
the error is half in French but I copied it above

**Desktop (please complete the following information):**

| Questions                                | Answers
|-----------------------------------------|--------------------
| Type of issue                            | Bug
| OS version (server)                   | Ubuntu (OVA file -> MONARC_v2.12.2@c27e4c8572489b762f667b28d42c5c1626bfe017)
| OS version (client)                    | Win11
| PHP version                              | 7.0, 7.1...
| MONARC version / git hash     | 2.12.2
| Browser                                    | Firefox, Chrome, Safari

**Comment by ruslanbaidan** [comment]
Hello,

You are trying to import a referential, but use a menu for the entire analysis import.
We have a particular import for this in the [Knowledge Base](https://www.monarc.lu/documentation/user-guide/#management-of-knowledge-base).  You can directly import it from mosp or download and import from file.
I attached a screenshot.
![Screenshot 2022-07-21 at 16 36 06](https://user-images.githubusercontent.com/3246171/180241382-07cbeb3c-1cc0-4075-a0bf-a86a83b36ed9.png)
